package.json

```diff
@@ -67,9 +67,7 @@
     "ws": "^0.8.0",
     "memory-streams": "^0.1.0",
     "dev-null": "^0.1.1",
-    "express": "^4.9.0"
-  },
-  "devDependencies": {
+    "express": "^4.9.0",
```

Comment:
shrinkwrap in node < 5 is broken w.r.t. devDependencies. We're just avoiding using them until a project is on node 5.